esmvaltool/cmorizers/obs/cmorize_obs.py

```diff
@@ -225,6 +225,9 @@ def _assemble_datasets(raw_obs, obs_list):
     tiers = [
         tier for tier in tiers if os.path.exists(os.path.join(raw_obs, tier))
     ]
+    if not tiers:
+        logger.warning("Could not find any Tier1,2... dir in raw data %s dir",
+                       raw_obs)
```

Comment:
Could you make this error message a bit nicer?

```suggestion
        logger.warning("Could not find any data directory matching %s/Tier{1,2,3}/",
                       raw_obs)
```

Reply:
No idea how this change is visible in this PR. It is the same as master, I didn't type it, and I also cannot remove it because the file is locked.

Reply:
Alright, that's odd... I see that it is the same as in master... No worries then 👍

Reply:
I merged master again and now it's gone... something strange happened there 😕 .